mlflow/server/js/src/model-registry/components/model-list/ModelListFilters.tsx

```diff
@@ -62,7 +62,7 @@ export const ModelSearchInputHelpTooltip = ({
           <FormattedMessage defaultMessage="Examples:" description="Text header for examples of mlflow search syntax" />
           <br />
           • tags.my_key = "my_value"
-          <br />• name ilike "%{exampleEntityName}%" and tags.my_key = "my_value"
+          <br />• name ilike "%{exampleEntityName}%" AND tags.my_key = "my_value"
```

Comment:
since we're fixing capitalization should we also do ILIKE?

```suggestion
          <br />• name ILIKE "%{exampleEntityName}%" AND tags.my_key = "my_value"
```